src/conjuring/visibility.py

```diff
@@ -1,8 +1,10 @@
+import re
 from pathlib import Path
 from typing import Callable
 
 from invoke import Task
 
+POETRY_LINE = re.compile(r"\[tool\..*poetry\]")
```

Comment:
> (not sure if it varies between most recent versions, hence regex)

I think a simple string match would work.
I don't think the section name changed.

But I'm fine with a regex.
It's not that it will slow down things too much.

Reply:
It's probably overengineering it, true. Literal string match would be more grep-friendly and easier to read too